js/src/util/abi.js

```diff
@@ -147,3 +149,31 @@ export function parseAbiType (type) {
   // If no matches, return null
   return null;
 }
+
+/**
+ * Returns a valid ABI by filter out interfaces
+ * with wrong typ
+ */
+export function toValidAbi (_abi = []) {
+  try {
+    return new Abi(_abi);
+  } catch (error) {
+    // If there is a wrong type, filter it out
+    if (/Cannot convert .+ valid ParamType/i.test(error.message)) {
+      const validInterfaces = [];
+
+      _abi.forEach((interf) => {
+        try {
+          Abi.parseABI([ interf ]);
```

Comment:
When doe this become a problem and why is this ont on the actual ABI layer instead of just benefitting the UI?